flixel/graphics/frames/FlxBitmapFont.hx

```diff
@@ -635,4 +646,51 @@ class FlxBitmapFont extends FlxFramesCollection
 		font.updateSourceHeight();
 		return font;
 	}
+
+	/*
+	 * Adds additional glyphs to this font.
+	 * 
+	 * @param   Frames             Additional glyphs frames.
+	 * @param   StartingCharacter  Starting character code for additional glyphs.
+	 */
+	public function appendGlyphs(Frames:FlxFramesCollection, StartingCharacter:Int = 500):Void
+	{
+		specialStart = StartingCharacter;
+		
+		var offY:Int = parent.height;
+		
+		var tmpBmp:BitmapData = new BitmapData(Math.ceil(Math.max(parent.width, Frames.parent.width)), parent.height + Frames.parent.height, true, 0x00000000);
+		tmpBmp.copyPixels(parent.bitmap, parent.bitmap.rect, point);
+		tmpBmp.copyPixels(Frames.parent.bitmap, Frames.parent.bitmap.rect, new Point(0, parent.height));
+		
+		parent = FlxGraphic.fromBitmapData(tmpBmp, parent.key + "+" + Frames.parent.key, true);
```

Comment:
this is a big part of it, with multiatlases we can simply take frames from one atlas and add them to another without needing to create a new bitmap containing every frame

Also line 666 breals flash CI, by skipping the `unique` arg